MIGRATION_PLAN.md

```
@@ -0,0 +1,698 @@
+   # ファクトリ関数（バリデーション付き）
+   @validate_call
+   def {type_name}(value: Annotated[{base_type}, Field({constraints})]) -> {type_name}:
+       return NewType('{type_name}', {base_type})(value)
+   ```
+
+   使用例:
+   ```python
+   user_id = {type_name}("user_12345")  # ✅ バリデーション実行
+   ```
+   """
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)